orderer/consensus/etcdraft/chain_test.go

```diff
@@ -3243,7 +3243,7 @@ var _ = Describe("Chain", func() {
 						})
 				})
 
-				It("aborts waiting for block to be committed upon leadership lost", func() {
+				FIt("aborts waiting for block to be committed upon leadership lost", func() {
```

Comment:
Because of this Focused testcase, other testcases are skipped. is there any reason for disabling other testcases.

Reply:
That’s not from me, must be from earlier commits I used. That’s an awesome find, maybe we just need to revert that line.